### ISSUE TITLE
Wsi > xml support

### ISSUE THREAD

**Issue by iixatomxii** [Foxxyyy/CodeX.Games.RDR1#12]
When changing any xml exported from wsi, the game will crash. Is there a method to get this to work that I’m missing, or is it a WIP? For example, if I open the xml and only change rotation of an object & save it, the game will crash. Rather it’s through codex saving or exporting the file and editing with n++ or other. I assume it may get support in the future, but I was curious if I was missing something to make it work, or if it’s just not supported yet

**Comment by Foxxyyy** [comment]
Yeah I need to fix these WSI, like other files... Though it's not really a priority for me right now. I'll keep you updated.

**Comment by iixatomxii** [comment]
Thank you for the reply! I totally understand and will patiently wait. Hell, even if it’s a year or more from now, better late than never. Thank you for adding RDR1 support regardless! I look forward to seeing this grow
